wpuf-functions.php

```diff
@@ -4012,3 +4012,34 @@ function wpuf_payment_success_page( $data ){
 
     return $redirect_page;
 }
+
+/**
+ * Remove conditional from form builder for selected fields
+ *
+ * @param $settings
+ *
+ * @return array
+ */
+function wpuf_unset_conditional( $settings ) {
+    $remove_cond_field = [ 'action_hook', 'step_start' ];
+
+    $field_settings = array_map(
+        function ( $field ) use ( $remove_cond_field ) {
+            if ( in_array( $field['template'], $remove_cond_field, true ) ) {
+                $index = array_filter(
+                    $field['settings'], function ( $settings ) {
+                    return $settings['name'] === 'wpuf_cond';
+                }
+                );
```

Comment:
Fix the indentation, and also resolve conflicts.

Reply:
Some PR has conflicts that will resolve after the previous one merges. otherwise, if we resolve this it will conflict again after merging the previous one.